feature-libs/asm/components/customer-selection/customer-selection.component.ts

```diff
@@ -42,6 +44,9 @@ export class CustomerSelectionComponent implements OnInit, OnDestroy {
 
   @ViewChild('resultList') resultList: ElementRef;
   @ViewChild('searchTerm') searchTerm: ElementRef;
+  @ViewChildren('resultItem') resultItems: QueryList<ElementRef<HTMLElement>>;
+
+  activeIndex = -1;
```

Comment:
If I understand correctly -
either change the name to make this more clear (`activeFocusedButtonIndex`) or leave a comment explaining this is for ARIA.

Reply:
ok will do tomorrow ^^